brayns/io/CircuitLoader.cpp

```diff
@@ -38,9 +39,11 @@ namespace brayns
 class CircuitLoader::Impl
 {
 public:
-    Impl(const ApplicationParameters& applicationParameters,
+    Impl(AnimationParameters& animationParameters,
```

Comment:
Consider moving the animationParameters out of here because this is not related to model loading, but attaching a model to the scene.